### ISSUE TITLE
defaultLimit in blueprints config don't works with value = 0

### ISSUE THREAD

**Issue by elrancid** [balderdashy/sails#3224]
In file node_modules/sails/lib/hooks/blueprints/actionUtils.js, at lines 33 and 284 there are:
```javascript
...
  populateEach: function ( query, req ) {
    var DEFAULT_POPULATE_LIMIT = sails.config.blueprints.defaultLimit || 30;
...
  parseLimit: function (req) {
    var DEFAULT_LIMIT = sails.config.blueprints.defaultLimit || 30;
...
```
I want to disable the limit. If I set "defaultLimit: 0" in blueprints config the "0" value is not get (it is considered as "false"), so "DEFAULT_POPULATE_LIMIT" and "DEFAULT_LIMIT" are set to 30 and the limit is not disabled.
I think this is wrong because if I call "find().limit(0)", it seems to works correctly and the limit is disabled.
For now, I've seen that I can disable the limit in the blueprints config setting "defaultLimit: -1". To be able to disable the limit in config with the "0" value you need to change the two line (33 and 284).

**Comment by sgress454** [comment]
Hi @elrancid, good catch--mind submitting a PR that fixes it?  With a test in `router.blueprintOptions.test.js`?

**Comment by elrancid** [comment]
Please, can someone explain me how to make the test?
I've branched the repo, made the changes, find the `router.blueprintOptions.test.js` file, go to the root, made `npm install` to install the dependencies and wrote `npm test`. The test don't seem to use the `router.blueprintOptions.test.js` (and made an error) and I don't know how to do to make the `router.blueprintOptions.test.js` file
My branched repo is at https://github.com/elrancid/sails/

**Comment by sailsbot** [comment]
Thanks for posting, @elrancid.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
